docs/one-click-integrations/jira-integration.md

```diff
@@ -1,29 +1,94 @@
+
+-   Pulse considers an incident every Jira issue in a **Done** status assigned with the label `Incident` (case-insensitive). **Done** status in Jira are all the status of an issue under the [Done category](https://support.atlassian.com/jira-work-management/docs/workflows-and-statuses-for-the-board/) (represented by the green color in Jira). Some examples are **DONE**, **CLOSED**, or **DECLINED**, but other values can be defined.
+
+-   Pulse associates an incident to one or more systems matching the values in the **Component(s)** field of the Jira issue (case-sensitive).
+
+    -   If the Jira issue is set with more than one component, Pulse creates an incident for each existing system that matches a component name, ignoring the remaining components that don't exist in Pulse as a system.
+
+    -   If the **Component(s)** field in the Jira issue is empty, or none of the components exist in Pulse as a system, Pulse creates an incident and associates it with the system `_unknown_`.
+
+-   The incident creation date is the timestamp when the Jira issue was created, while the incident resolution date is the timestamp when the Jira issue was last updated to a **Done** status.
+
+-   Pulse creates new incidents when the following updates are performed in Jira:
+
+    -   The status of an issue assigned with the label `Incident` is updated to a **Done** status. Pulse creates one or more incidents, depending on the values in the **Component(s)** field.
+
+    -   An issue in a **Done** status is assigned with the label `Incident`. Pulse creates one or more incidents, depending on the values in the **Component(s)** field.
+
+    -   A new component that matches an existing system is added to an issue in a **Done** status assigned with the label `Incident`. Pulse creates a new incident associated with the matching system.
 
-1.  Wait until you get a confirmation that Pulse successfully created the integration and the necessary webhooks on Jira.
+-   Pulse deletes an existing incident when the following updates are performed in Jira:
 
-    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+    -   The corresponding issue is re-opened.
+
+    -   The label `Incident` is removed from the corresponding issue.
+
+    -   A component that matches an existing system is removed from the corresponding issue. Pulse deletes the incident for that system.
+
+    -   The corresponding issue is deleted.
+
+!!! important
+    After completing the Jira integration setup, Pulse starts loading your incident data for the last 90 days. Therefore, **before you perform the integration setup**, make sure the corresponding Jira issues follow the rules described above, so Pulse can load your historical data correctly.
+    Pulse will only create incidents for the Jira issues in a **Done** status that are assigned with the label `Incident`, and associates them with the systems matching the values in the **Component(s)** field as described above.
```

Comment:
This looks like it would be useful as an introduction before the instructions. I'm also adding a couple of tweak suggestions:
* present tense
* "assigned with the label" sounds odd. How about "assigned a label" or the even simpler "labeled"?

I wonder if it would make sense to move this further up before the instructions (and tweak it slightly for context):

```suggestion
!!! important
    After completing the Jira integration setup, Pulse starts loading your incident data for the last 90 days. Therefore, **before you perform the integration setup**, make sure the corresponding Jira issues follow the rules described below, so Pulse can load your historical data correctly.
    Importantly, Pulse only creates incidents for Jira issues that are in a **Done** status and labeled `Incident`, and associates them with the systems matching the values in the **Component(s)** field as described below.
```

Reply:
@nicklem Thank you for this suggestion. However, I'm not sure if the flow for the reader will work this way...
I need more time to check on this. I'm closing this comment for now and I'll get back to it after the launch. 👍